src/directory/directory.js

```diff
@@ -674,6 +674,11 @@ const directory = {
             route: '/lib/storage/download',
             filters: ['android', 'ios', 'flutter', 'js', 'react-native']
           },
+          {
+            title: 'Query Transfers',
+            route: '/lib/storage/querytransfers',
+            filters: ['android', 'ios', 'flutter', 'js', 'react-native']
```

Comment:
```suggestion
            filters: ['android']
```
looks like only Android is supported on this page